sympy/series/limits.py

```diff
@@ -319,8 +319,8 @@ def remove_abs(expr):
 
         try:
             if str(dir) == '+-':
-                r = gruntz(e, z, z0, '+')
-                l = gruntz(e, z, z0, '-')
+                r = limit(e, z, z0, '+')
+                l = limit(e, z, z0, '-')
                 if l != r:
                     raise ValueError("The limit does not exist since "
```

Comment:
A simple check could be
If r == Limit(e, z, z0, '+') or l == Limit(e, z, z0, '+') :
Return Limit(e, z, z0, '+-')

Reply:
Or other could be
If isinstance(r, Limit) or isinstance(l, Limit):
Return Limit(e, z, z0, '+-')